src/actions/guides/command_line_tasks/built_in.cr

```diff
@@ -14,44 +14,46 @@ class Guides::CommandLineTasks::BuiltIn < GuideAction
     You can think of a Luck Task just like a [Rake task](https://github.com/ruby/rake) in Ruby, [npm script](https://docs.npmjs.com/misc/scripts) for Node, or [Mix Task](https://hexdocs.pm/mix/Mix.Task.html) for Elixir.
 
     To run a task, you just run
-    ```plaintext
+    ```plain
```

Comment:
These updates were caused from https://github.com/luckyframework/website/pull/579 no longer including the `plaintext`.